package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc -p .",
     "watch": "tsc -w",
     "lint": "eslint --fix .",
-    "test": "mocha -r ts-node/register test/*.ts",
+    "test": "npm run build && mocha -r ts-node/register test/*.ts",
```

Comment:
Can we use `ts-node` command to test CLI of typescript directly instead of building before every test?

There is a script `command` in package.json as below.
https://github.com/unvt/charites/blob/c586c707b151f561decacfcfe301ed1e0a5ff805/package.json#L14

Reply:
maybe, it requires to change to `./node_modules/.bin/ts-node ./src/cli.ts`in package.json